### DISCUSSION TITLE
In my Github images not open

### DISCUSSION THREAD

**Discussion by kownblog** [community/community#59980] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In my account images not showing what problems are coming can anyone solve it?
![img](https://github.com/community/community/assets/43713657/fadd7a45-99ec-4d3b-8a71-a77a4f3bf119)

**Comment by 3urobeat** [comment]
This looks like a browser/internet related issue on your end. Have you tried visiting the same page using a different browser?

**Reply by kownblog** [reply]
Now i can check images are open
